lib/edition_dependencies_populator.rb

```diff
@@ -5,8 +5,8 @@ def initialize(edition)
   end
 
   def populate!
-    @edition.depended_upon_contacts = Govspeak::ContactsExtractor.new(@edition.body).contacts
-    @edition.depended_upon_editions = Govspeak::DependableEditionsExtractor.new(@edition.body).editions
+    @edition.depended_upon_contacts = Govspeak::ContactsExtractor.new(@edition.body).contacts.uniq
```

Comment:
wouldn't it make the `ContactsExtractor` and `DependableEditionsExtractor` responsible for producing unique lists?